ksqldb-parser/src/test/java/io/confluent/ksql/parser/KsqlParserTest.java

```diff
@@ -427,12 +427,6 @@ public void testSelectAll() {
     assertThat(query.getSelect().getSelectItems(), is(contains(new AllColumns(Optional.empty()))));
   }
 
-  @Test
-  public void testReservedColumnIdentifers() {
-    assertQuerySucceeds("SELECT ROWTIME as ROWTIME FROM test1 t1;");
-    assertQuerySucceeds("SELECT ROWKEY as ROWKEY FROM test1 t1;");
-  }
```

Comment:
What's the issue with this test? Why remove it?

Reply:
There was a Rowkey resolved issues that I resolved